lib/gooddata/rest/phmap.rb

```diff
@@ -7,8 +7,16 @@
 module GoodData
   module Rest
     class Connection
+
+      def self.map_placeholders(title)
+        @@PH_MAP.each do |pm|
+          break if title.gsub!(pm[1], pm[0])
+        end
+        title
+      end
+
       # PH_MAP for wildcarding of URLs in reports
-      PH_MAP = [
+      @@PH_MAP = [
```

Comment:
Style/ClassVars: Replace class var @@PH_MAP with a class instance var.

Reply:
A Ruby constant would be more suitable for storing the mapping.